### DISCUSSION TITLE
Using it makes ngDoCheck() keep firing rapidly

### DISCUSSION THREAD

**Discussion by alpha93** [MurhafSousli/ngx-scrollbar#358] [Q&A] [answered]
Is this normal and is there a way to stop this from happening?

**Comment by MurhafSousli** [comment]
`ngDoCheck` works properly, and it does not fire when you scroll the scrollbar [stackblitz](https://stackblitz.com/edit/ngx-scrollbar-vtdecx?file=src%2Fapp%2Fapp.component.html)

**Comment by MurhafSousli** [comment, marked as answer]
The issue was only if the default change detection strategy, was caused by updating the scrollbar on `ngAfterViewChecked`, this is fixed version v7.6.0 and higher

related #373
